components/admin_console/admin_sidebar/admin_sidebar.tsx

```diff
@@ -277,7 +274,7 @@ class AdminSidebar extends React.PureComponent<Props, State> {
     }
 
     isPluginPresentInSections = (plugin: PluginRedux) => {
-        return this.state.sections?.indexOf(`plugin_${plugin.id}`) !== -1;
+        return this.state.sections && this.state.sections.indexOf(`plugin_${plugin.id}`) >= 0;
```

Comment:
The `non-null` check can stay here since we changed the comparison from `strict non-equal` to `bigger-or-equal-to` and therefore `null` no longer produces false-positives

```suggestion
        return this.state.sections?.indexOf(`plugin_${plugin.id}`) >= 0;
```

Reply:
@michelengelen The reason I changed: when I use this approach I still get a warning: 
![image](https://user-images.githubusercontent.com/37421564/171002552-b36561f1-1d5d-4c2e-916f-dfe506a305d2.png)

I think I have two options:
- ```return this.state.sections && this.state.sections.indexOf(`plugin_${plugin.id}`) >= 0```
- ```return this.state.sections?.indexOf(`plugin_${plugin.id}`) !== -1```

Reply:
Wow, thats interesting. I tested this on the TS-Playground before and it works without complaining about it being possibly undefined ... well, it is working with the double check, so you can just keep it as is.